ui/components/package.json

```diff
@@ -18,7 +18,8 @@
     "clean": "rimraf dist/",
     "build": "tsc --build",
     "build:cjs": "tsc --project ./tsconfig.cjs.json",
-    "test": "echo 'no test to run' && exit 0",
+    "test": "TZ=UTC jest",
+    "test:watch": "TZ=UTC jest --watch",
```

Comment:
it would be cool if this script is added in other package where the tests are actually running like in `panels-plugin`